client/state/data-layer/wpcom/read/tags/test/utils.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import { fromApi } from 'state/data-layer/wpcom/read/tags/utils';
+
+const successfulFollowedTagsResponse = {
+	tags: [
+		{
+			ID: '307',
+			slug: 'chickens',
+			title: 'Chickens',
+			display_name: 'chickens',
+			URL: 'https://public-api.wordpress.com/rest/v1.2/read/tags/chickens/posts',
+		},
+		{
+			ID: '148',
+			slug: 'design',
+			title: 'Design',
+			display_name: 'design',
+			URL: 'https://public-api.wordpress.com/rest/v1.2/read/tags/design/posts',
+		},
+	]
+};
+
+const normalizedFollowedTagsResponse = [
+	{
+		id: '307',
+		slug: 'chickens',
+		title: 'Chickens',
+		displayName: 'chickens',
+		url: '/tag/chickens',
+	},
+	{
+		id: '148',
+		slug: 'design',
+		title: 'Design',
+		displayName: 'design',
+		url: '/tag/design',
+	},
+];
+
+const successfulSingleTagResponse = {
+	tag: {
+		ID: '307',
+		slug: 'chickens',
+		title: 'Chickens',
+		display_name: 'chickens',
+		URL: 'https://public-api.wordpress.com/rest/v1.2/read/tags/chickens/posts'
+	},
+};
+
+const normalizedSuccessfulSingleTagResponse = [
+	{
+		id: '307',
+		slug: 'chickens',
+		title: 'Chickens',
+		displayName: 'chickens',
+		url: '/tag/chickens',
+	},
+];
+
+describe( 'wpcom-api: read/tags utils', () => {
+	describe( '#fromApi', () => {
+		it( 'should properly normalize many tags', () => {
+			const transformedResponse = fromApi( successfulFollowedTagsResponse );
```

Comment:
do you want to deepfreeze your fixtures? would guard against tests mutating them.